tests/integration/qualification/loadbalancing_test.go

```diff
@@ -54,70 +52,69 @@ const (
 // To use a particular kubeconfig (other than ~/.kube/config), set:
 // --istio.test.kube.config=<path>
 func TestIngressLoadBalancing(t *testing.T) {
```

Comment:
@nmittler should we remove this test? its not actually run

Reply:
Yeah I'm fine removing all of the `qualification` tests ... the intention was noble, but it never gained any traction.